quotes/common.js

```diff
@@ -18,7 +44,7 @@ function createPopup(userkey, roles, userdata) {
     avatar.alt = "avatar for " + userkey
 
     // An optional tag, attached to the name of the user (such as for bots)
-    var tagSpan = null
+    let tagSpan = null;
```

Comment:
Does the semicolon matter? Just curious, since it's present in a fair few places from your changes.

Reply:
Nope semicolons shouldn't (in most cases), IDE quickfix probably added them, I'll clean these up